android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetActivity.java

```diff
@@ -81,6 +94,10 @@ public void onDestroy() {
         // current meeting, but when our view is detached from React the JS <-> Native bridge won't
         // be operational so the external API won't be able to notify the native side that the
         // conference terminated. Thus, try our best to clean up.
+        getJitsiView().dispose();
+
+        JitsiMeetActivityDelegate.onHostDestroy(this);
```

Comment:
This (both statements) should be done right before `super.onDestroy` or there is no chance for the code in between to run because the RN glue will have been destroyed.